zerver/data_import/rocketchat.py

```diff
@@ -238,18 +239,92 @@ def convert_huddle_data(
     return zerver_huddle
 
 
+def build_custom_emoticons(
```

Comment:
Emoticons are something different in our terminology; this should be `build_custom_emoji`.